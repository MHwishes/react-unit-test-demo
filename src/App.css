.app {
    text-align: center;
}

.title {
    color: rgba(175, 47, 47, 0.15);
    font-size: 73px;
}

/*input.new-todo {*/
    /*padding: 16px 16px 16px 60px;*/
    /*border-color: rgba(0, 0, 0, 0.03);*/
    /*width: 100%;*/
    /*background-color: white;*/
/*}*/
