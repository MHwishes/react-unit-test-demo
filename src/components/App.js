import React from 'react'
import Footer from './Footer'
import AddTodo from './AddTodo'
import VisibleTodoList from './TodoList'
import '../App.css'

const App = () => (
    <div className="todoapp">
        <h1 className="title">Todos</h1>
        <div className="main">
            <AddTodo/>
            <VisibleTodoList/>
        </div>
        <Footer/>
    </div>
);

export default App
