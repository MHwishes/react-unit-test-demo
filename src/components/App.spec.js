import React from "react";
import {shallow} from 'enzyme';
import App from './App'
import toJson from 'enzyme-to-json';

describe("<App/>", () => {
    it('render app component structure', () => {
        const wrapper = shallow(<App/>);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
