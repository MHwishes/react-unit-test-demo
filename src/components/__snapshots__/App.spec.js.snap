// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> render app component structure 1`] = `
<div
  className="todoapp"
>
  <h1
    className="title"
  >
    Todos
  </h1>
  <div
    className="main"
  >
    <Connect(AddTodoView) />
    <Connect(TodoList) />
  </div>
  <Footer />
</div>
`;

exports[`shallow render App component render app component structure 1`] = `
<div
  className="todoapp"
>
  <h1
    className="title"
  >
    Todos
  </h1>
  <div
    className="main"
  >
    <Connect(AddTodoView) />
    <Connect(TodoList) />
  </div>
  <Footer />
</div>
`;
